tribe-ext-use-newer-core-js/index.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar Extension: Use Newer Core JS
+ * Description:     Loads more current versions of core scripts like jQuery and Underscore when viewing various admin pages of The Events Calendar and Event Tickets.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__Use_Newer_Core_JS
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+/**
+ * Extension main class, class begins loading on init() function.
+ */
+class Tribe__Extension__Use_Newer_Core_JS extends Tribe__Extension {
+
+
+	/**
+	 * Setup the Extension's properties. This always executes even if the required plugins are not present.
+	 *
+	 * @since 1.0.0
+	 */
+	public function construct() {
+		$this->add_required_plugin( 'Tribe__Events__Main' );
+		$this->set_url( 'https://theeventscalendar.com/extensions/use-newer-core-js/' );
+		$this->set_version( '1.0.0' );
```

Comment:
This isn't needed for the initial version. Just mentioning that in case you weren't aware, no real harm in having it either it's just not of much use.